python/nav/portadmin/snmputils.py

```diff
@@ -396,7 +396,7 @@ def set_cisco_voice_vlan(self, interface, voice_vlan):
         """Should not be implemented on anything else than Cisco"""
         raise NotImplementedError
         
-    def set_cisco_cdp(self, interface, action):
+    def enable_cisco_cdp(self, interface, action):
```

Comment:
Now I noticed the `action` argument here, which is not present in the overriding method in the Cisco subclass (not keeping the method signature is frowned upon). What is this supposed to be used for?

Reply:
The `action` argument there is a relic which can be omitted. Just pushed a new commit.